index.js

```diff
@@ -120,6 +123,9 @@ function extendContext(context, opts) {
       },
     },
     session: {
+      commit() {
```

Comment:
I think this is incorrect here if you want to add a `commit` method in `ctx.session`.

also if we really add this method, it may be a breaking change because maybe somebody's has code like `if (ctx.session.commit) { /* do something */}`. maybe we should use a more special method name or only add this method when `autoCommit` set to false.